pystove/version.py

```diff
@@ -14,4 +14,4 @@
 # along with pystove.  If not, see <http://www.gnu.org/licenses/>.
 #
 # Copyright 2019 Milan van Nugteren
-__version__ = '0.2a1'
+__version__ = '0.3a1'
```

Comment:
Please split out the version bump to a separate PR.

Reply:
> Please split out the version bump to a separate PR.

Why?

Reply:
Each PR should be as small as possible and fulfill a single purpose. See [PR best practices](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/best-practices-for-pull-requests#write-small-prs)

Reply:
Actually the increment of the minor version reflects the "feature" addition and is directly related to the changes of this PR. So IMHO it is totally reasonable to increase it along  with adding the feature. If you insist to remove it I will do so.

Reply:
By that logic, adding 3 new features in quick succession between 2 releases should increase the minor version by 3 as well...
As it is now this PR adds a feature AND increases the version number, fulfilling more than 1 purpose and making it larger than necessary (although still pretty small in general ;) ).

Reply:
Which are the 2 other features? The devcontainer PR should not increase the library version as it is just about the development environment. It changes nothing in the library source code.

Anyway. I'm going to remove it. At the end you are in charge to do the release. I'm crossing my fingers that I don't have to wait for long 😉